.buildkite/hooks/environment

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# export PATH="/.pulumi/bin:$PATH"
+# export PULUMI_ACCESS_TOKEN="$(buildkite-agent secret get PULUMI_ACCESS_TOKEN)"
+
+# eval "$(pulumi env open cnunciato/default/personal --format shell)"
```

Comment:
Leaving this here just for reference, as I suspect this is where we'll want to continue setting the required environment variables.